watertap/unit_models/zero_order/tests/test_dual_media_filtration_zo.py

```diff
@@ -0,0 +1,176 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Tests for zero-order dual media filtration model
+"""
+import pytest
+
+from io import StringIO
+from pyomo.environ import (
+    ConcreteModel, Constraint, value, Var, assert_optimal_termination)
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import initialization_tester
+
+from watertap.unit_models.zero_order import DualMediaFiltrationZO
+from watertap.core.wt_database import Database
+from watertap.core.zero_order_properties import WaterParameterBlock
+
+solver = get_solver()
+
+class TestDualMediaFiltrationZO:
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.db = Database()
+
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+        m.fs.params = WaterParameterBlock(
+            default={"solute_list": ["nonvolatile_toc", "toc", "tss"]})
+
+        m.fs.unit = DualMediaFiltrationZO(default={
+            "property_package": m.fs.params,
+            "database": m.db})
+
+        m.fs.unit.inlet.flow_mass_comp[0, "H2O"].fix(10)
+        m.fs.unit.inlet.flow_mass_comp[0, "nonvolatile_toc"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "toc"].fix(1)
+        m.fs.unit.inlet.flow_mass_comp[0, "tss"].fix(1)
+
+        return m
+
+    @pytest.mark.unit
+    def test_build(self, model):
+        assert model.fs.unit.config.database == model.db
+
+        assert isinstance(model.fs.unit.electricity, Var)
+        assert isinstance(model.fs.unit.energy_electric_flow_vol_inlet, Var)
+        assert isinstance(model.fs.unit.electricity_consumption, Constraint)
+
+    @pytest.mark.component
+    def test_load_parameters(self, model):
+        data = model.db.get_unit_operation_parameters("dual_media_filtration")
+
+        model.fs.unit.load_parameters_from_database()
```

Comment:
@luohezhiming - you might want to add another test when using `load_parameters_from_database(use_default_removal=True)` to ensure that a default removal rate is fixed for solutes that are not listed in the yaml file.

Reply:
Should I add `load_parameters_from_database(use_default_removal=True)` and check all parameters again? Or just set `use_default_removal=True` to this line 67?

Reply:
I think you could do something similar to the testing done in `test_nanofiltration_zo.py` and other unit model test files where you add a new set of tests for a solute_list that includes a component that doesn't exist in the yaml file.

Reply:
OK. I figure out what I missed, I should add a test class with `use_defalt_removal=True` . Will do.

Reply:
I've added the test for the case with extra solute.